hw/ip/rom_ctrl/dv/formal/rom_ctrl_fpv.core

```diff
@@ -0,0 +1,20 @@
+CAPI=2:
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+name: "lowrisc:dv:rom_ctrl_fpv:0.1"
+description: "ROM_CTRL formal verification"
+filesets:
+  files_formal:
+    depend:
+      - lowrisc:ip:rom_ctrl
+    files:
+      - tb.sv
+    file_type: systemVerilogSource
+
+targets:
+  formal:
+    default_tool: icarus
```

Comment:
Is Jasper called Icarus in FuseSoC or is that a separate tool?

Reply:
No, it's just a placeholder as a "bog standard tool that takes a file list". If you grep for "icarus" in the codebase you'll find other equivalent hits.